LinuxBuild OpenSUSE missing dependency lua-devel

In the build instructions for openSUSE (*/readme/LinuxBuild.md*), the `lua-devel` dependency is missing.

If `lua-devel` is not installed, there are build abortions with `g++` in certain situations (s. *Steps to Reproduce* below). 
On OpenSUSE Tumbleweed, `lua-lgi` does not automatically install `lua-devel`. Therefore, one must manually install it. In the commands for Fedora & Solus, for instance, the `lua-devel` dependency is included.

**Solution**
Add `lua-devel` to the *Distribution specific commands for openSUSE* in */readme/LinuxBuild.md*. 

**Steps to Reproduce**
System:
- OpenSUSE Tumbleweed
- Kernel: `5.18.2-1-default`
- Use the in *LinuxBuild.md* provided command to install the dependencies:

```
sudo zypper install cmake gtk3-devel portaudio-devel libsndfile-devel \
texlive-dvipng texlive libxml2-devel libpoppler-glib-devel libzip-devel librsvg-devel lua-lgi \
gtksourceview4-devel
```

- Try to build from commit `cc4e3e8312ed9a1dd8fe808b03a31ff75a8cd0d9`
```
(...)
[cmake] -- Could NOT find Lua (missing: LUA_LIBRARIES LUA_INCLUDE_DIR) (Required is at least version "5.3")
(...)
[build] [274/318  86% :: 47.692] Building CXX object src/core/CMakeFiles/xournalpp-core.dir/undo/InsertDeletePageUndoAction.cpp.o
[build] ninja: build stopped: subcommand failed.
[build] Build finished with exit code 1
```
- Install `lua-devel` & build again
```
[build] Build finished with exit code 0
```

--- a/readme/LinuxBuild.md
+++ b/readme/LinuxBuild.md
@@ -53,7 +53,7 @@ libgtksourceview-4-dev
 #### For openSUSE:
 ```bash
 sudo zypper install cmake gtk3-devel portaudio-devel libsndfile-devel \
-texlive-dvipng texlive libxml2-devel libpoppler-glib-devel libzip-devel librsvg-devel lua-lgi \
+texlive-dvipng texlive libxml2-devel libpoppler-glib-devel libzip-devel librsvg-devel lua-devel lua-lgi \
 gtksourceview4-devel
 ```
 

